javascript/Face/sdk_quickstart.js

```diff
@@ -42,7 +42,7 @@ async function DetectFaceExtract() {
 	await Promise.all (image_file_names.map (async function (image_file_name) {
```

Comment:
It seems not all languages in the QS have quality filter logic for enrollment and recognition?

Reply:
Correct. I'm not comfortable committing sample code in both languages that I am unfamiliar with and that are currently untestable due to the client updates still being in progress. We could revisit once the client update happens for them.